src/test/regress/specs/isolation_citus_dist_activity.spec

```diff
@@ -0,0 +1,98 @@
+setup
```

Comment:
We may add some tests using `master_run_from_same_connection` UDF defined in #2363 to check whether we observe the transactions initiated from workers.